translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"\t\tsymlinks, devices, pipes, etc)."
+
+#: pkg/kubectl/cmd/create_namespace.go:32
+msgid ""
+"\n"
+"\t\tCreate a namespace with the specified name."
+msgstr ""
+"\n"
+"\t\tCria um namespace com um nome especificado."
+
+#: pkg/kubectl/cmd/create_secret.go:119
+msgid ""
+"\n"
+"\t\tCreate a new secret for use with Docker registries.\n"
+"\n"
+"\t\tDockercfg secrets are used to authenticate against Docker registries.\n"
+"\n"
+"\t\tWhen using the Docker command line to push images, you can authenticate "
+"to a given registry by running\n"
+"\n"
+"\t\t    $ docker login DOCKER_REGISTRY_SERVER --username=DOCKER_USER --"
+"password=DOCKER_PASSWORD --email=DOCKER_EMAIL'.\n"
+"\n"
+"    That produces a ~/.dockercfg file that is used by subsequent 'docker "
+"push' and 'docker pull' commands to\n"
+"\t\tauthenticate to the registry. The email address is optional.\n"
+"\n"
+"\t\tWhen creating applications, you may have a Docker registry that requires "
+"authentication.  In order for the\n"
+"\t\tnodes to pull images on your behalf, they have to have the credentials.  "
+"You can provide this information\n"
+"\t\tby creating a dockercfg secret and attaching it to your service account."
+msgstr ""
+"\n"
+"\t\tCria um novo secret para utilizar com o Docker registries.\n"
+"\n"
+"\t\tDockercfg secrets são utilizados para autenticar no Docker registries.\n"
+"\n"
+"\t\tQuando utilizando a linha de comando do Docker para realizar push das "
+"images, você pode se autenticar para um registro fornecido executando\n"
+"\n"
+"\t\t    $ docker login DOCKER_REGISTRY_SERVER —username=DOCKER_USER —"
+"password=DOCKER_PASSWORD —email=DOCKER_EMAIL’.\n"
+"\n"
+"    Isso irá gerar um arquivo ~/.dockercfg que será utilizado para os "
+"comandos ’docker push’ e ‘docker pull’ \n"
+"\t\tse autenticarem no registro. O endereço de email é opcional.\n"
+"\n"
+"\t\tQuando criar aplicações, você pode ter uma Docker registry que requer "
+"autenticação.  Para que \n"
+"\t\tos nodes possam realizar o pull das imagens em seu nome, eles devem ter "
+"as credenciais.  Você pode prover esta informação\n"
+"\t\tcriando um dockercfg secret e anexando-o à sua conta de serviço."
+
+#: pkg/kubectl/cmd/create_pdb.go:32
+msgid ""
+"\n"
+"\t\tCreate a pod disruption budget with the specified name, selector, and "
+"desired minimum available pods"
+msgstr ""
+"\n"
+"\t\tCria um pod disruption budget com o nome especificado, seletor, e o "
+"número mínimo de pode disponíveis"
+
+#: pkg/kubectl/cmd/create.go:42
+msgid ""
+"\n"
+"\t\tCreate a resource by filename or stdin.\n"
+"\n"
+"\t\tJSON and YAML formats are accepted."
+msgstr ""
+"\n"
+"\t\tCria um recurso por nome de arquivo ou stdin.\n"
+"\n"
+"\t\tOs formatos JSON e YAML são aceitos."
+
+#: pkg/kubectl/cmd/create_quota.go:32
+msgid ""
+"\n"
+"\t\tCreate a resourcequota with the specified name, hard limits and optional "
+"scopes"
+msgstr ""
+"\n"
+"\t\tCria um resourcequota com o nome especificado, hard limits e escopo "
```

Comment:
I'm really thinking if we have a better word here for "hard limits", but "limites físicos" or anything else seems strange. Ideas?